messenger.rst

```diff
@@ -844,6 +844,12 @@ options.
 AMQP Transport
 ~~~~~~~~~~~~~~
 
+.. versionadded:: 5.2
+
+    Starting from Symfony 5.2, the AMQP transport can handle AMQPS DSN (e.g.  ``amqps://``).
+    Be aware that using it without using CA certificate can throw an exception.
+    An alternative is to use AMQP DSN and to specify which port you want to use (e.g.  ``amqp://localhost?port=5671``).
```

Comment:
We often avoid introducing new information in versionadded directives. When the version reached end-of-life, we simply remove all old versionadded directives - it would be sad to loose usefull information.

So I would suggest you say this information somewhere below in the text. I think the best is to add a real example to the `.env` example on line 866, e.g.:

```
# .env
MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages

# or use the AMQPS protocol
MESSENGER_TRANSPORT_DSN=amqps://...
```

You can then also show a working DNS string (so people see how to configure the required CA certificate). After the example, you can then add a short versionadded directive:

```rst
.. versionadded:: 5.2

    Support for the AMQPS DSN was introduced in Symfony 5.2.
```

Reply:
Wdyt about the changes ?